Finalising SRS and readme for Demo 1

iwandejong opened the pull request:
Please review the PDF in particular

Xavier893 reviewed (changes requested):
- Class Diagram of System is needed for SRS.
- Will confirm about the use case diagrams, but I think one use case diagram per use case implemented is needed
- Third use case for CRUD needed

iwandejong commented:
> * Class Diagram of System is needed for SRS.
> 
>     * Will confirm about the use case diagrams, but I think one use case diagram per use case implemented is needed
> 
>     * Third use case for CRUD needed

Changes have been made. Please review

iwandejong commented:
- Added problem statement to introduction
- Added service contracts outlining the functional requirements
- Added descriptions to the Backend Class Diagram to mention Design Patterns
- Made Architectural Pattern more generic (excluded technologies used)
